components/script/dom/gpudevice.rs

```diff
@@ -349,7 +365,10 @@ impl GPUDevice {
             },
             multiview: None,
         };
-        Ok((implicit_ids, desc))
+        Ok((
+            PipelineLayout::Implicit(pipeline_layout.implicit().unwrap()),
```

Comment:
The `unwrap` here will panic whenever the pipeline_layout is `Explicit`, whereas previously it would have simply returned `None`. I think we just need to return the `pipeline_layout` from line 260.